src/modules/Nfs.ycp

```diff
@@ -352,6 +353,60 @@
 	return true;
     }
 
+    /*
+     * Supports(a, b) is true if a must be mounted before b.
+     * That is, "/" Supports "/any", "/foo" Supports "/foo/bar",
+     * but "/foo" does not Support "/foof"
+     */
+    boolean Supports(any entry_a, any entry_b) {
+      if (! is(entry_a, map)) return false;
+      if (! is(entry_b, map)) return false;
+      map map_a = (map) entry_a;
+      map map_b = (map) entry_b;
+      string a = map_a["file"]:"";
+      string b = map_b["file"]:"";
+
+      if (a == b) return false;
+
+      boolean StartsWith(string s, string head) {
```

Comment:
Yes, `String.StartsWith` exists but `String.EndsWith` does not, so since the functions are trivial anyway I preferred consistency.